.github/workflows/package_for_test.yml

```diff
@@ -1,11 +1,31 @@
 name: Package Neuron for Test
 
 on: 
-  pull_request:
+  issue_comment:
+    types: [created, edited]
   push:
```

Comment:
I thought that `package for test` would be passive and triggered by a comment on demand. The `push` event here makes each commit packaged, that makes `pacakge by comment` redundant.

Reply:
The package action trigger by push existed before. So I didn't remove it.

Reply:
But `package by comment` will be redundant if each commit has its own package

Reply:
Most of the time, we will not directory push to `Neuron`, we push to our fork repo. `package by comment` is used to package for the pr branch.

Reply:
> Most of the time, we will not directory push to `Neuron`, we push to our fork repo. `package by comment` is used to package for the pr branch.

Got it